__tests__/__utils__/kv.ts

```diff
@@ -30,11 +30,9 @@ export function createKV<Key extends string>(): KV<Key> {
     // eslint-disable-next-line @typescript-eslint/require-await
     async put(key: Key, value: string, _?: { expirationTtl: number }) {
       values[key] = value
-      expect(key.length).toBeLessThanOrEqual(512)
+      if (key.length > 512) {
+        throw 'Error: key longer than 512 characters.'
```

Comment:
```suggestion
        throw new Error('Error: key longer than 512 characters.')
```

Always throw an instance of `Error` instead of other objects.